packages/eslint-plugin-foreman/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "@theforeman/eslint-plugin-foreman",
+    "version": "3.2.0",
```

Comment:
Please use the latest version `3.4.0` (then the `bot` will change it to `3.5.0` after the merge